tools/isobuild/meteor-npm.js

```diff
@@ -749,6 +773,8 @@ function isSubtreeOf(subsetTree, supersetTree, predicate) {
     }
   }
 
+  console.log("\nNot subtree:", JSON.stringify(subsetTree, null, 4), JSON.stringify(supersetTree, null, 4), "\n");
```

Comment:
And these "Not object"/"Not subtree" logs also very nebulous. Also "\n" at the end adds double newlines, probably unneeded. Also, we should probably be using `runLog.log`, like the rest of this file.